app/components/LoginForm/LoginPage.tsx

```diff
@@ -1,8 +1,10 @@
+import { Card } from '@webkom/lego-bricks';
 import AuthSection from 'app/components/AuthSection/AuthSection';
 import { Content } from 'app/components/Content';
 
 const LoginPage = () => (
   <Content>
+    <Card severity="danger">Denne siden krever innlogging</Card>
     <AuthSection />
   </Content>
 );
```

Comment:
Maybe rename this component?  I know we don't normally use a dedicated Login Page like this, but since this is a component / page that should only be shown when someone attempts to access a page that requires auth, the name could reflect that?